great_expectations/core/metric_domain_types.py

```diff
@@ -0,0 +1,11 @@
+import enum
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class MetricDomainTypes(enum.Enum):
```

Comment:
```suggestion
class MetricDomainTypes(str, enum.Enum):
```

Should this use a `str` mixin?

Reply:
@Kilo59 How would we utilize this mixin?